### PR TITLE
Bump haproxy http-request timeout to 30 seconds

### PR DESCRIPTION
**Pull request by cybertron**
Per the request in our discussion with the api team, we want to try
increasing this timeout to see if it helps with our api flakiness.

Note that I did not change the openstack platform's timeout because
they were already using 1 minute.

**- Description for the changelog**
Increase the http-request timeout in haproxy on platforms that use that to loadbalance the api.

### CODE REVIEW

**Comment by cybertron** [comment]
/retest

This didn't solve the problem we were having, but it was suggested by the api team so it's probably still a good idea to do it.

**Comment by kikisdeliveryservice** [comment]
@cybertron do you still need this PR open, if not could you close?

**Comment by cybertron** [comment]
/retest

Failed on unrelated image pull error.

**Comment by cybertron** [comment]
/retest

This doesn't affect any of the failing platforms so I'm not sure we need to block on them.

**Comment by cybertron** [comment]
/remove-lifecycle stale

I think we still want to get this in to be in sync with the loadbalancer settings on other platforms.

**Comment by cybertron** [comment]
This passed all of the relevant jobs. I believe we need approval from all of the on-prem platforms now.

/cc @mandre @Gal-Zaidman @jcpowermac @patrickdillon

**Review by mandre — APPROVED** [review]
/lgtm

**Comment by cybertron** [comment]
Hmm, I might need to rebase this to pick up the owners change. It's still wanting MCO approval.

**Review by mandre — APPROVED** [review]
/lgtm

**Comment by mandre** [comment]
> Hmm, I might need to rebase this to pick up the owners change. It's still wanting MCO approval.

Still after the rebase :confused:

**Comment by cybertron** [comment]
I'll look into it. It looks like we lost the ci jobs for the other platforms too.

/test e2e-openstack
/test e2e-ovirt
/test e2e-vsphere

**Comment by kikisdeliveryservice** [comment]
remove the hold once you're satisfied with the tests

/approve
/hold

**Comment by cybertron** [comment]
To close the loop on this, the on-prem groups are only reviewers (not approvers) on the on-prem directory since we can't all approve at once. So this is behaving as intended.

**Comment by cybertron** [comment]
/retest

That went poorly. ;-)

**Comment by kikisdeliveryservice** [comment]
ok let's just retest this..

/retest

**Comment by mandre** [comment]
/retest
C'mon e2e-openstack, you can do it

**Review by mandre — COMMENTED** [review]
/lgtm

**Comment by cybertron** [comment]
We have ci passes from all of the affected platforms and lgtm from 3/4. Since this is highly unlikely to break anything, much less an individual platform, I think we can proceed.

**Pull request merged**
